lib/help.js

```diff
@@ -11,6 +11,7 @@ module.exports = chalk`
   {bold Options}
     {cyan --help, -h} this output
     {cyan --json, -h} output as json
+    {cyan --all} outputs detailed report of support audit
```

Comment:
I would recommend this to be 
`--verbose/-v`
OR
`--details/-d`

Reply:
`--verbose` makes sense and meets with a lot of conventions that already exist.